openmc/element.py

```diff
@@ -217,9 +226,10 @@ def expand(self, percent, percent_type, enrichment=None,
         if enrichment is not None:
 
             # Calculate the mass fractions of isotopes
-            abundances['U234'] = 0.008 * enrichment
+            abundances['U234'] = 0.0089 * enrichment
             abundances['U235'] = enrichment
-            abundances['U238'] = 100.0 - 1.008 * enrichment
+            abundances['U236'] = 0.0046 * enrichment
```

Comment:
This strikes me as high and looking at the cited report I couldn't find any commentary or citations on where they got the relative isotopic ratios. Do you know if these isotopic ratios are accurate for centrifuge-enriched natural uranium? I trust what has been done in SCALE and VERA, but I was just a little concerned since I didn't seen any justification in the cited report.

Reply:
To be honest, I don't know. My general thought process was the same (if it's in SCALE/VERA, surely someone has checked it). If you can find out from some of your colleagues down there, I'd love to know though!

Reply:
Ok, I'll ask around down here and let you know what I hear.

Reply:
I did a bit more digging on the 236U content of natural uranium. [The Isotopic compositions of the elements 2013 (IUPAC Technical Report)](https://www.degruyter.com/downloadpdf/j/pac.2016.88.issue-3/pac-2015-0503/pac-2015-0503.pdf) does not list the isotopic abundance of 236U, but the reference cited from the report, [Isotopic "fingerprints" for natural uranium ore samples](http://www.sciencedirect.com/science/article/pii/S1387380699001025), seems pretty reliable and notes that an upper limit for n(236U)/n(238U) of about 6e-10 was found in their samples. I'm inclined to think that the Yankee core test fuel assembly where these isotopic ratios came from contains some reprocessed uranium or their numbers are not correct. I'll ask the CASL folks first thing in the morning to see what they think.

Reply:
When you enrich Uranium in U235, you actually end up enriching U234 and U236 as well, so I'm not sure how much the natural abundances would really tell you. That being said, you make a good point that if the fuel was downblended, the isotopics could be different than fuel that was normally enriched.